integration-test-standard/src/test/java/com/ericsson/bss/cassandra/ecaudit/integration/custom/ITVerifyPostLogging.java

```diff
@@ -176,34 +177,70 @@ private String replaceBatchIdWithConstant(String s)
     }
 
     @Test
-    public void testFailedStatements()
+    public void testFailedSimpleStatements()
```

Comment:
The idea here is that the original test case "testFailedStatements" is divided in 3 parts, where only the first part is applicable in the setup we have in 2.2. As we merge this into 3.0.11 we should remove the @Ignore on the two remainging test cases.